src/util/certs/get-certs.ts

```diff
@@ -12,7 +12,10 @@ export default async function getCerts(
   client: Client,
   domain?: string
 ) {
-  const query = domain ? stringify({ domain }) : '';
+  const query = stringify({
+    ...domain ? { domain } : {},
+    limit: 20
+  });
```

Comment:
If we make a second request we can find out whether the response set was capped and print an appropriate message. Do you think it's worth it? We don't have a general plan for pagination in the CLI yet.